.github/workflows/ci.yaml

```diff
@@ -0,0 +1,48 @@
+name: Node.js CI
+
+on:
+  pull_request:
+    branches: [ "develop" ]
+
+jobs:
+  lint:
+    runs-on: ubuntu-latest
+
+    steps:
+    - uses: actions/checkout@v3
+    - name: Use Node.js
+      uses: actions/setup-node@v3
+      with:
+        node-version: '18.x'
+        cache: 'npm'
```

Comment:
This line wont work properly unless you provide a `cache-dependency-path`. You can skip it for now as we will implement a more robust caching strategy later.